llvm/include/llvm/SandboxIR/Use.h

```diff
@@ -0,0 +1,52 @@
+#ifndef LLVM_TRANSFORMS_SANDBOXIR_USE_H
+#define LLVM_TRANSFORMS_SANDBOXIR_USE_H
+
+#include "llvm/IR/Use.h"
+#include "llvm/Support/raw_ostream.h"
+
+namespace llvm {
+namespace sandboxir {
```

Comment:
`namespace llvm::sandboxir {`

Reply:
Maybe not. `llvm::Use` could be forward declared. IIRC `raw_ostream` can also be forward declared.

Reply:
To get rid of the include.

Reply:
Including llvm IR header files is not going to cause any dependency issues, so I think it should be fine to keep it this way.